locale/coa/ar/Personal.xml

```diff
@@ -0,0 +1,269 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration xmlns="http://ledgersmb.org/xml-schemas/configuration">
```

Comment:
Shouldn't the configuration be in the project? We are using ledgersmb.org as a CDN.

Reply:
The URL for a schema need not hold a document, but first and foremost needs to be unique. Using our own domain as the basis guaranteed that we are the only ones using the namespace.

Reply:
The configuration *is* in the project, really: https://github.com/ledgersmb/LedgerSMB/blob/master/doc/company-setup/configuration.xsd

Reply:
(Could you mark this conversation "resolved" if that answers your question/addresses your point?)